Source/8_UI/Replayer/Components/Toolbar/Timeline.cs

```diff
@@ -153,6 +153,9 @@ internal class Timeline : ReeUIComponentV2, IReplayTimeline {
         }
 
         private void GenerateDefaultMarkersFromReplay(IReplay replay) {
+            float firstNoteTime = replay.NoteEvents.First().eventTime;
```

Comment:
Use `FirstOrDefault` instead of `First`. Using `First` will cause exception when `NoteEvents` is null.

Reply:
Same with `Last`.